prow/jobs/rafter/rafter.yaml

```diff
@@ -1,142 +1,279 @@
-job_labels_template: &job_labels_template
+presets:
+  - labels:
+      preset-rafter-build-master: "true"
+    env:
+      - name: BUILD_TYPE
+        value: master
+      - name: DOCKER_PUSH_DIRECTORY
+        value: ""
+  - labels:
+      preset-rafter-minio-gcs-gateway: "true"
+    env:
+      - name: MINIO_GATEWAY_MODE
+        value: gcs
+      - name: CLOUDSDK_COMPUTE_ZONE
+        value: "europe-west4-a"
+  - labels:
+      preset-rafter-minio-az-gateway: "true"
+    env:
+      - name: MINIO_GATEWAY_MODE
+        value: azure
+      - name: AZURE_RS_GROUP
+        value: rafter-minio-gateway
+      - name: AZURE_REGION
+        value: westeurope
+
+common_job_labels_template: &common_job_labels_template
   preset-dind-enabled: "true"
   preset-kind-volume-mounts: "true"
+
+common_job_labels_push_template: &common_job_labels_push_template
   preset-sa-gcr-push: "true"
   preset-docker-push-repository-kyma: "true"
 
-test_infra_ref: &test_infra_ref
-  org: kyma-project
-  repo: test-infra
-  base_ref: master
-  path_alias: github.com/kyma-project/test-infra
+common_job_labels_minio_gcs_gateway_template: &common_job_labels_minio_gcs_gateway_template
+  preset-rafter-minio-gcs-gateway: "true"
+  preset-sa-gke-kyma-integration: "true"
 
-common_config: &common_config
-  skip_report: false
-  always_run: true
+common_job_labels_minio_az_gateway_template: &common_job_labels_minio_az_gateway_template
+  preset-rafter-minio-az-gateway: "true"
+  preset-creds-aks-kyma-integration: "true"
+
+additional_postsubmit_job_labels_template: &additional_postsubmit_job_labels_template
+  prow.kyma-project.io/slack.skipReport: "true"
+
+common_job_config_template: &common_job_config_template
   decorate: true
-  optional: false
   path_alias: github.com/kyma-project/rafter
   max_concurrency: 10
-
-job_build_template: &job_build_template
-  <<: *common_config
   extra_refs:
```

Comment:
Now that I think about it we should pin that repo, test-infra, to specific commit, what do you think?

Reply:
I agree with you, but we must consider to which. I suggest to Friday last commit.

Reply:
To latest available :)